apps/bare-expo/ios/Podfile.lock

```diff
@@ -548,7 +548,7 @@ PODS:
   - React-jsinspector (0.71.0)
   - React-logger (0.71.0):
     - glog
-  - react-native-netinfo (9.3.5):
+  - react-native-netinfo (9.3.7):
```

Comment:
@gabrieldonadel lockfile update for netinfo is here (https://github.com/expo/expo/pull/20913)